_includes/asg/gcp-template.md

```diff
@@ -16,8 +16,8 @@ See [GCP disk](https://cloud.google.com/compute/docs/disks/) for more details on
 
 Examples:
 
-* `"type=pd-ssd,size=200"`
-* `"type=pd-standard,size=200","type=pd-ssd,size=100"`
+* `"-s", "type=pd-ssd,size=200"`
```

Comment:
for k8s, people will read this page to figure out what to fill in for install.portworx.com. Wouldn't the `-s` confuse them?

Reply:
Currently we do not have any field that sets the max_drive_set_count on the installer page. May be only the examples for limiting storage less nodes I can provide that flag and make  a note that this needs to be done in the Args field of the daemonset

Reply:
Yeah looks good now.